server/routes/api/chart.py

```diff
@@ -378,3 +378,164 @@ def geojson(dcid):
     }),
                     200,
                     mimetype='application/json')
+
+
+def get_choropleth_sv():
+    """ Gets all the stat vars and accompanying denominators that will have a choropleth chart drawn for them
+    ie. charts with the hasChoropleth property set as true
+
+    Returns: 
+        a tuple consisting of:
+            set of all stat vars including both the stat vars of the charts and stat vars of denominators
+            dictionary of stat var: list of denominators
+    """
+    chart_config = current_app.config['CHART_CONFIG']
+    if os.environ.get('FLASK_ENV') == 'development':
```

Comment:
suggest to remove line 393-395. We have seen a bug from chart config but was not able to catch it in dev :)

Might just need to restart the server when chart config is changed, which is not that bad